src/content/docs/style-guide/writing-guidelines/user-related-language-guidelines.mdx

```diff
@@ -72,13 +72,29 @@ We recommend linking to the [standard roles](/docs/accounts/accounts-billing/new
 [Default groups](/docs/accounts/accounts-billing/new-relic-one-pricing-users/users-roles#user-group)      </td>
 
       <td>
-"When adding a user, add them to the [**Admin** group](/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure/#groups)."
+Example use: 
+*When adding a user, you can add them to either the [**Admin** or **User** group](/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure/#groups).*
  </td>
 
       <td>
-Sentence case, bold.
+Preferred: sentence case, bold. Alternatively, instead of bold, you can use italics or code-formatting: as long as it's clear that we're referring to something that has a definite, exact name. The words 'admin' and 'user' are generic, general words, so using formatting to indicate that we're referencing specific terms is important. 
+
```

Comment:
For this and similar recs throughout, it necessary to provide this many alternatives? I feel like a better strategy might be to propose a current best practice and be more directive about sticking with that.